opm/autodiff/StandardWellsDense.hpp

```diff
@@ -86,21 +86,22 @@ enum WellVariablePositions {
             typedef typename GET_PROP_TYPE(TypeTag, MaterialLaw)         MaterialLaw;
             typedef typename GET_PROP_TYPE(TypeTag, Simulator)           Simulator;
             typedef typename GET_PROP_TYPE(TypeTag, IntensiveQuantities) IntensiveQuantities;
+            typedef typename GET_PROP_TYPE(TypeTag, Scalar)              Scalar;
 
-            typedef double Scalar;
             static const int numEq = BlackoilIndices::numEq;
-            static const int numWellEq = GET_PROP_VALUE(TypeTag, EnablePolymer)? 3:numEq; // //numEq; //number of wellEq is only for 3 for polymer
+            static const int numWellEq = GET_PROP_VALUE(TypeTag, EnablePolymer)? numEq-1 : numEq; // //numEq; //number of wellEq is only numEq for polymer
```

Comment:
I think this fix is correct, but the comment is wrong, should be (numEq - 1) I think.